src/test/cli.up.test.ts

```diff
@@ -33,6 +33,15 @@ describe('Dev Containers CLI', function () {
 			await shellExec(`docker rm -f ${containerId}`);
 		});
 
+		it('should execute successfully with valid config and dotfiles', async () => {
+			const res = await shellExec(`${cli} up --workspace-folder ${__dirname}/configs/image --dotfiles-repository https://github.com/devcontainers/dotfiles`);
+			const response = JSON.parse(res.stdout);
+			assert.equal(response.outcome, 'success');
```

Comment:
Can we add asserts to validate if the dotfiles changes are reflected?

Reply:
Added